src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -155,17 +155,25 @@ public enum ErrorCode {
   VECTORIZECONFIG_CHECK_FAIL("Internal server error: VectorizeConfig check fail"),
 
   UNAUTHENTICATED_REQUEST("UNAUTHENTICATED: Invalid token"),
-  INVALID_QUERY("Invalid query"),
-  DATABASE_READ_FAILED("Database read failed"),
-  DRIVER_TIMEOUT("Driver timeout"),
-  DRIVER_CLOSED_CONNECTION("Driver request connection is closed"),
-  NO_NODE_AVAILABLE("No node was available to execute the query"),
-  NO_INDEX_ERROR("Faulty collection (missing indexes). Recommend re-creating the collection"),
   COLLECTION_CREATION_ERROR(
       "Collection creation failure (unable to create table). Recommend re-creating the collection"),
   INVALID_SCHEMA_VERSION(
       "Collection has invalid schema version. Recommend re-creating the collection"),
-  INVALID_ID_TYPE("Invalid Id type");
+  INVALID_ID_TYPE("Invalid Id type"),
+
+  // Driver failure codes
+  /** Error codes related to driver exceptions. */
+  SERVER_NO_NODE_AVAILABLE("No node was available to execute the query"),
+  SERVER_INVALID_QUERY("Invalid query"),
```

Comment:
Isn't this a user error? According to the error mapping code we return 200 OK. Should remove `SERVER_` prefix here.